examples/cassandra/README.md

```diff
@@ -53,54 +53,48 @@ This is a somewhat long tutorial.  If you want to jump straight to the "do it no
 In Kubernetes, the atomic unit of an application is a [_Pod_](../../docs/user-guide/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.
 In this simple case, we define a single container running Cassandra for our pod:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-controller.yaml -->
+<!-- BEGIN MUNGE: EXAMPLE cassandra.yaml -->
 
 ```yaml
 apiVersion: v1
-kind: ReplicationController
+kind: Pod
```

Comment:
Have you read issue #19098? The readme for the [cassandra example](https://github.com/kubernetes/kubernetes/tree/master/examples/cassandra) starts by deploying a single pod (instead of a rc) but the code for the pod is incorrect (it's an rc). Just read the issue and the readme...